### ISSUE TITLE
Include category: founders

### ISSUE THREAD

**Issue by eugelr** [graphprotocol/mission-control-curator#306]
I would like to have a trusted resource to know the identity of the founders of crypto projects. This would help me get more credibility. One good example in this field are DeFi projects and Staking providers where you have to send your funds. In those cases, having a "face" to put to the project and a real person would be very benefial to increase trust in crypto.

**Comment by yanivtal** [comment]
Yes **People** should be its own registry and we should link **Projects** to the team members.
